client/src/featureform/register.py

```diff
@@ -2479,6 +2479,45 @@ def df_transformation(self,
         self.__resources.append(decorator)
         return decorator
 
+    def ondemand_feature(self,
+                          tags: List[str],
+                          properties: dict,
```

Comment:
Do we want to make these default arguments so users don't have to specify tags and properties?

Reply:
addressed this in local mode PR